.github/workflows/main.yml

```diff
@@ -9,12 +9,16 @@ permissions:
   contents: read
 
 jobs:
-  build:
+  jar-build:
     runs-on: ubuntu-latest
+
     permissions:
       contents: read
       packages: write
```

Comment:
I don't think we need packages here anymore as we're building image in a separate job

Reply:
Agreed.

Removed in [76ed720](https://github.com/giom-l/kafka-ui/commit/76ed720828c409376ba55785e7e4f286cb6ba2aa)